### ISSUE TITLE
lost in its layout

### ISSUE THREAD

**Issue by gedw99** [PixyBlue/lazy-lob-web#1]
just being honest but git sub modules makes it really hard to follow.

You could just do  a mono repo pattern and use .gitignore to not commit the code of the sub repos.
I do it and its stupid simple..

**Comment by taophp** [comment]
Got it ! You're right ! I'll do that .

**Comment by taophp** [comment]
Done! https://github.com/PixyBlue/lazy-lob-web/commit/85bb3b0c86e2f22bfb46a0b3d9311ed6652e7fcf
